### PR TITLE
RMT Sub-chapter modification.

### PR DESCRIPTION
**Pull request by farrosalferro**
The following changes are added:
1. RetNet's code implementation and its overview.
2. Official repo of RetNet, RMT, and ViR.

### CODE REVIEW

File: chapters/en/Unit 13 - Outlook/Retention.mdx [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+
+The overall architecture of ViR is quite similar to that of ViT, except that it replaces the Multi-Head Attention (MHA) with Multi-Head Retention (MHR). This MHR mechanism is free of any gating function and can be switched between parallel, recurrent, or chunkwise (a hybrid between parallel and recurrent) modes. Another difference in ViR is that the positional embedding is first added to the patch embedding, and then the [class] token is appended.
+
+## PyTorch Implementation of Multi-Scale Retention
+The implementation of Multi-Scale Retention (main proposal in RetNet) from the original paper is shown below:
+```python
+import torch
+import torch.nn.functional as F
+from torch import nn
+from .rms_norm import RMSNorm
```
Comment (mmhamdy) [inline chapters/en/Unit 13 - Outlook/Retention.mdx]: I think we need to have an independent code snippet. These imports from the original code will be confusing to readers.

**Review by mmhamdy — COMMENTED** [review]
Hi @farrosalferro. This is super nice! Thanks again for your work. My only concern is that the provided code is dependent on code from the original implementation.

File: chapters/en/Unit 13 - Outlook/Retention.mdx (hunk 1 above)
Comment (farrosalferro) [reply chapters/en/Unit 13 - Outlook/Retention.mdx]: I see, I'll add the rest of the code so that it becomes independent. Thanks for the suggestion!

Comment (mmhamdy) [reply chapters/en/Unit 13 - Outlook/Retention.mdx]: Sounds cool!

**Pull request merged**